### ISSUE TITLE
EEF1E1 human and mouse

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-annotation#1504]
http://www.uniprot.org/uniprot/O43324

http://www.uniprot.org/uniprot/Q9D1M4

I'll come back to this one, there are a few things to adress...

**Comment by pgaudet** [comment]
@ValWood 
Can you please elaborate ? and assign/tag someone. 

Thanks, Pascale

**Comment by ValWood** [comment]
where to begin !

http://www.uniprot.org/uniprot/O43324
negative regulation of cell proliferation Source: UniProtKB
positive regulation of apoptotic process Source: UniProtKB
positive regulation of cellular senescence Source: BHF-UCL
positive regulation of DNA damage response, signal transduction by p53 class mediator Source: UniProtKB
tRNA aminoacylation for protein translation Source: Reactome

http://www.uniprot.org/uniprot/Q9D1M4
embryo development Source: UniProtKB
negative regulation of cell proliferation Source: UniProtKB
positive regulation of apoptotic process Source: UniProtKB
positive regulation of apoptotic signaling pathway Source: MGI
positive regulation of cellular senescence Source: MGI
positive regulation of DNA damage response, signal transduction by p53 class mediator

this is elongation factor 

look like readouts.....

**Comment by pgaudet** [comment]
These are very downstream/indirect/uncharacterized relations. 
This is *exactly* for these cases that I want to implement the relation 'causally upstream or involved in'. People find these correlations, which are definitely interesting, but for which we don't have any mechanistic explanation.

Pascale

**Comment by pgaudet** [comment]
@ggeorghiou 
@hdrabkin 
@RLovering 
Can you have a look at the annotations, and see if they need revision? 

Thanks, Pascale

**Comment by ggeorghiou** [comment]
@pgaudet @ValWood I've reread the paper and the author's intent is to show that p18 elongation factor does indeed interact with ATR and play a role in apoptosis and cell cycle. The negative regulation of cell proliferation comes from interactions via p21/p53 as does apoptotic signaling. I don't think these need revisions. But a qualifer term like 'causally upstream or involved in' would be ideal in this situation to better clarify the role it has.

**Comment by ValWood** [comment]
OK thanks will close. I thought it was one of the many things which get activated to regulation of a checkpoint but are really just causing a pathological problem that is detected by the checkpoint and cannot be corrected, subsequently activates cell death pathway. This one appears to have a moonlighting function in that case. Sorry about that, I should have checked the paper.

**Comment by rachhuntley** [comment]
Commenting for our cellular senescence annotation, which I had already looked at before seeing this has been closed.
EEF1E1 is also called Aminoacyl tRNA synthetase complex-interacting multifunctional protein 3 (AIMP3) and this paper ("miR-543 and miR-590-3p regulate human mesenchymal stem cell aging via direct targeting of AIMP3/p18”) describes a novel role of AIMP3 in cellular aging, the whole paper is about the role of this protein in senescence. 
In the first sentence they say "Previously, AIMP3 (aminoacyl-tRNAsynthetase-interacting multifunctional protein-3) was shown to be involved in the macromolecular tRNA synthetase complex or to act as a tumor suppressor. In this study, we report a novel role of AIMP3/p18 in the cellular aging of human mesenchymal stem cells (hMSCs).” With this and the fact the protein is describes as multi-functional, I felt confident in annotating it to this term.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4259092/

**Comment by ValWood** [comment]
I closed this one last night.

However I still feel some of these are not normal processes. For example
"AIMP3/p18 overexpression is sufficient to induce the cellular senescence phenotypes"

So, these might just be checkpoint activating problems,  which would induce senescence if the problem cannot be overcome? I guess this is one of the problems in studying tumour cells and one which can be addressed more clearly in yeast.

By analogy,  here we found ~500 fission yeast gene products cause problems that when mutated activate a checkpoint and affect cell cycle progression https://www.ncbi.nlm.nih.gov/pubmed/23697806
Now have over  1000 fission  yeast genes  with mutants which cause problems that when mutated activate a checkpoint and affect cell cycle progression (cause a mitotic arrest phenotype of some sort).  
However those working on these genes/phenotypes do not consider these genes to be "part of" the checkpoint,  or even "regulating the checkpoint", or the regulating cell cycle.   Inducing "cell cycle arrest" is not their normal function. These mutant alleles, or expression levels, are inducing a problem that the checkpoint detects. "Cell cycle phenotype" is not equivalent to "cell cycle process". 

I felt that most of the experiments could be explained  similarly. Here, I was swayed by the fact that p18 also shown to binds to ATM  (although translation elongation factor EF-1 gamma subunit is one of the most highly expressed protein in a yeast cell  >33,000 copies ),  and these are  routine contaminants in  protein most  protein preps.....so even that seemed uncertain. I could not see controls to show that their mutant  had not affected protein levels or the amount of uncomplexed protein.

So personally,  although I agree these are " ageing related phenotypes" I think it is very unclear if this is "normal biology" , and something we should think about in the future.  Ageing is tricky anyway, probably almost  everything probably contributes  to ageing eventually  (inability to deal with metabolic defects or  inability to deal with cell cycle defects, the biggest  problem kills you). 

Because we don't have a good cell level phenotype ontology, and we try to shoehorn everything abnormal into GO....... "causally upstream"  won't help because it isn't "normal", and will make things even unclearer .....

Anyway this is a whole big discussion and will probably become clearer with time......
